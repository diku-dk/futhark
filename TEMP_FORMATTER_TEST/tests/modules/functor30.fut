-- Based on #1741
-- ==
-- error: type t = i64

module Op = (\(X: {type t = i64 val x : t}) -> {def x = X.x})
            : (X: {type t val x: t}) -> {val x : X.t}

module L2 = Op {
  type t = bool
  def x : t = true
}

def main = L2.x
