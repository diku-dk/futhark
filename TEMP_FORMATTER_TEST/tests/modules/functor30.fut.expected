3764e011b8efbef655706957f58214af
